package.json

```diff
@@ -3,7 +3,7 @@
   "productName": "BetterDiscord Installer",
   "description": "A simple standalone program which automates the installation, removal and maintenance of BetterDiscord.",
   "author": "BetterDiscord",
-  "version": "1.1.1",
+  "version": "1.2.0",
```

Comment:
Please don't update the version

Reply:
Yeah that was mainly meant for my own branch's release number, but I figured it could be a problem for you. :) Will remove that commit.